src/frontend/src/lib/components/dapps/DappModal.svelte

```diff
@@ -49,9 +49,7 @@
 		{#if nonNullish(screenshots) && screenshots.length > 0}
 			<div class="overflow-hidden rounded-3xl">
 				<ImgBanner
-					fitHeight={true}
-					height="100%"
-					width="100%"
+					styleClass="h-64 max-h-64"
```

Comment:
if height is defined, do you need `max-h-`?

Reply:
the `max-h-` is needed but the `h-` is unnecessary because the image can shrink and does not have a fixes height